packages/smart-contracts/scripts/test-deploy-all.ts

```diff
@@ -18,4 +19,5 @@ export default async function deploy(_args: any, hre: HardhatRuntimeEnvironment)
   await deploySuperFluid(hre);
   await deployBatchConversionPayment(_args, hre);
   await deployERC20TransferableReceivable(_args, hre, mainPaymentAddresses);
+  await deploySingleRequestProxyFactory(_args, hre, mainPaymentAddresses);
```

Comment:
We deploy the SingleRequestProxyFactory *last* so it doesn't impact the `nonce` for the preceding deployments. Some fo the tests rely on these nonces.